api-docs/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "api-docs",
+  "version": "25.0.0-alpha8",
+  "private": true,
+  "description": "Astro-based documentation site for Vaadin web components API",
+  "license": "Apache-2.0",
+  "author": "Vaadin Ltd",
+  "type": "module",
+  "scripts": {
+    "dev": "astro dev",
+    "start": "astro dev",
+    "build": "astro build",
+    "preview": "astro preview",
+    "astro": "astro",
+    "generate": "tsx generate.ts"
```

Comment:
Used `tsx` for now. Since Node 23 this works without, but we don't have that on CI servers yet.